mocha.sass

```diff
@@ -342,6 +342,16 @@ body {
   margin: 0;
   color: var(--mocha-stats-color);
   z-index: 1;
+  background-color: #00e600;
+  border: solid  #00e600;
+  border-radius: 10px;
+}
+
+#mocha-stats.fail {
+ @extend #mocha-stats;
```

Comment:
`@extend` won't work now that we've removed sass. I personally don't have experience with sass but I'm good with CSS, I'll dig in and see if I can replace this

[From JoshuaKGoldberg on the original PR](https://github.com/mochajs/mocha/pull/4148/files/c9b2939c98789cf23bccfd17e23682601c7a8ca2#r1509367853):

[Style] I don't think this @extend is really needed. We can achieve the same result in raw CSS, such as with CSS variables or different class names.

Thinking in terms of semantic class names, I think it'd make sense to have a class like .fail, .pass, etc. that each define their own colors.